airbyte-integrations/connectors/source-github/unit_tests/test_stream.py

```diff
@@ -1381,9 +1381,9 @@ def test_stream_contributor_activity_accepted_response(caplog):
         status=202,
     )
     with patch("time.sleep", return_value=0):
-        list(read_full_refresh(stream))
```

Comment:
let's reproduce this bug in a test so we would not miss it again

Reply:
this test reproduces this "bug", None is not returning now from read records so read results now can be correctly parsed in https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/sources/utils/record_helper.py#L14.